tests/platform_tests/conftest.py

```diff
@@ -520,7 +520,17 @@ def post_reboot_analysis(marker, event_counters=None, reboot_oper=None, log_dir=
             # restore original script. If the ".orig" file does not exist (upgrade path case), ignore the error.
             duthost.shell("mv {} {}".format(reboot_script_path + ".orig", reboot_script_path),
                           module_ignore_errors=True)
-        result = loganalyzer.analyze(marker, fail=False)
+        # For mac jump test, the log message we care about is uaually combined with other messages in one line,
+        # which makes the length of the line longer than 1000 and get dropped by Logananlyzer. So we need to increase
+        # the max allowed length.
+        # The regex library in Python 2 takes very long time (over 10 minutes) to process long lines. In our test,
+        # most of the combined log message for mac jump test is around 5000 characters. So we set the max allowed
```

Comment:
The standard warm-reboo test (not the mac-jump case) also looks for MAC learning events. 

So, I think we should enable it for all advanced-reboot cases. But, that will bring up the time.
If we do not enable for all, then the check for no mac jumping is not reliable, as it can pass (due to issue being fixed here) even when mac jumping happens unexpectedly.

Reply:
Yes, exactly. We may miss the MAC jump event if we don't apply the same change to all advanced-reboot test cases, which leads to false pass.
Currently I limited the length to 6000. I didn't see noticeable time increase in `test_warm_reboot_mac_jump`. I think we can apply the change to all advanced-reboot test cases.